release/ray_release/glue.py

```diff
@@ -178,9 +179,13 @@ def run_release_test(
                 "session_timeout", DEFAULT_CLUSTER_TIMEOUT
             )
 
-            autosuspend_mins = test["run"].get("autosuspend_mins", None)
+            autosuspend_mins = test["cluster"].get("autosuspend_mins", None)
             if autosuspend_mins:
                 cluster_manager.autosuspend_minutes = autosuspend_mins
+            else:
+                cluster_manager.autosuspend_minutes = min(
+                    DEFAULT_AUTOSUSPEND_MINS, int(cluster_timeout / 60) + 10
+                )
```

Comment:
```suggestion
                cluster_manager.autosuspend_minutes = min(
                    DEFAULT_AUTOSUSPEND_MINS, int(command_timeout / 60) + 10
                )
```

I think this should be command timeout, right? Assuming that jobs are not recognized as running correctly

Reply:
oh yeah good catch